ci/scripts/run-kitchen.sh

```diff
@@ -0,0 +1,77 @@
+source ci/variables/env
+
+# Parse parameters
+while [[ "$#" > 0 ]]; do case $1 in
+  --module) MODULE="$2"; shift; shift;;
+  --tag) TAG="$2"; shift; shift;;
+  --aws-profile) AWS_PROFILE="$2"; shift; shift;;
+  --action) ACTION="$2"; shift; shift;;
+  --args) ARGS="$2"; shift; shift;;
+  *) usage "Unknown parameter passed: $1"; shift; shift;;
+esac; done
+
+# Default version of Terraform when not specified a tag
+TAG="${TAG:-$DEFAULT_KITCHEN_TERRAFORM_TAG}"
+IMAGE="quay.io/dwp/kitchen-terraform:${TAG}"
+
+# Default AWS Profile
+AWS_PROFILE="${AWS_PROFILE:-"default"}"
+
+printf "\n*************************************************************\n"
+printf "  Running Kitchen Terraform using the following parameters:\n"
+printf "  %-22s %s\n" "- Module:" ${MODULE:-"(not set)"}
+printf "  %-22s %s\n" "- Terraform version:" $TAG
+printf "  %-22s %s\n" "- AWS Profile:" $AWS_PROFILE
+printf "  %-22s %s\n" "- Kitchen action:" ${ACTION:-"(not set)"}
+printf "  %-22s %s\n" "- Kitchen arguments:" ${ARGS:-"(not set)"}
+printf "*************************************************************\n\n"
+
+if [[ -n "$MODULE" ]] && [[ -n "$ACTION" ]]; then
+
+  if [[ ${ACTION} == "debug" ]]; then
+    docker run -ti --rm \
+      --env pipeline_id=$PIPELINE_ID \
+      --env AWS_PROFILE=$AWS_PROFILE \
+      --env CUSTOM_CA_DIR=/usr/share/ca-certificates/custom \
+      --volume /etc/ssl/certs/:/usr/share/ca-certificates/custom \
+      --volume $(pwd):/usr/action \
+      --volume ~/.ssh:/root/.ssh \
+      --volume ~/.aws:/root/.aws \
+      --volume ~/.terraformrc:/root/.terraformrc \
+      --user root \
+      --workdir /usr/action/ \
+      --entrypoint bash \
+      ${IMAGE}
+  elif [[ ${ACTION} == "verify" || ${ACTION} == "test" && ${ARGS} == *"hybrid-ecs"*  &&  -z ${KONG_EE_LICENSE} ]]; then
+    docker run -ti --rm \
+      --env pipeline_id=$PIPELINE_ID \
+      --env AWS_PROFILE=$AWS_PROFILE \
+      --env KONG_EE_LICENSE=$KONG_EE_LICENSE \
+      --env CUSTOM_CA_DIR=/usr/share/ca-certificates/custom \
+      --volume /etc/ssl/certs/:/usr/share/ca-certificates/custom \
+      --volume $(pwd):/usr/action \
+      --volume ~/.ssh:/root/.ssh \
+      --volume ~/.aws:/root/.aws \
+      --volume ~/.terraformrc:/root/.terraformrc \
+      --user root \
+      --workdir /usr/action/ \
+      --entrypoint bash \
+      ${IMAGE}
```

Comment:
This can be shortened to:

```suggestion
  if [[ ${ACTION} == "debug" ]]; then
    docker run -ti --rm \
      --env pipeline_id=$PIPELINE_ID \
      --env AWS_PROFILE=$AWS_PROFILE \
      --env CUSTOM_CA_DIR=/usr/share/ca-certificates/custom \
      $(if [[ ${ACTION}=="verify" || ${ACTION}=="test" && ${ARGS}==*"ecs"* && -n ${KONG_EE_LICENSE} ]]; then echo "--env KONG_EE_LICENSE=${KONG_EE_LICENSE}"; fi) \
      --volume /etc/ssl/certs/:/usr/share/ca-certificates/custom \
      --volume $(pwd):/usr/action \
      --volume ~/.ssh:/root/.ssh \
      --volume ~/.aws:/root/.aws \
      --volume ~/.terraformrc:/root/.terraformrc \
      --user root \
      --workdir /usr/action/ \
      --entrypoint bash \
      ${IMAGE}
```

Reply:
optimised the if/else statement